R/utils.R

```diff
@@ -6,8 +6,7 @@
 #' @param path The path within the `stan/` folder of the installed `epidist`
 #' package to the Stan code chunk of interest.
 #' @return A character string containing the Stan code chunk of interest.
-#' @family utils
-#' @export
+#' @noRd
```

Comment:
for all internal functions we might consider a rename. 1. They likely don't need the epidist prefix and 2. we might want to use common these are internal signals like . or _ as a prefix

Reply:
Happy with remove `epidist_`. Don't have major desire to do `.` or `_` but if there is a push I can do it

Reply:
The reason is to indicate these are internal functions vs not. Its generally pretty standard practice so I would favour doing so but I don't feel strongly

Reply:
For the `brms` ones I think that would break things

Reply:
[a3447be](https://github.com/epinowcast/epidist/pull/300/commits/a3447be829f7a91d62270020d08925bdbb1c7821) does the remove `epidist_` part

Reply:
> For the brms ones I think that would break things

Then don't do it for those ones. What is the push back for the others?

Reply:
It's just not something I'm familiar with standard practise on / I have some doubts about the extent to which it is standard practise. But OK as I say I don't mind so I'll add it

Reply:
"Have some doubts" --> pick an R package I think they would know what they are doing, then what proportion of the time are they doing this? For example I picked `ggplot2` and I don't see it there

Reply:
can't start with underscores - that's just a weird bit of R naming rules (https://cran.r-project.org/doc/FAQ/R-FAQ.html#What-are-valid-names_003f). I've definitely seen the starts-with-dot for internals.

Reply:
(it's probably leaning into the base R `.Internal()` as an emphasis idiom)

Reply:
I think done in [d45d0c9](https://github.com/epinowcast/epidist/pull/300/commits/d45d0c9afdd08711badba3e6212d6680706c05ab)